server/graphql/v2/input/AccountUpdateInput.js

```diff
@@ -0,0 +1,18 @@
+import { GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import { Currency } from '../enum/Currency';
+
+export const AccountUpdateInput = new GraphQLInputObjectType({
+  name: 'AccountUpdateInput',
+  fields: () => ({
+    id: {
+      type: GraphQLString,
```

Comment:
I'm wondering if we shouldn't make this `id` mandatory and the only way to reference the account for this mutation, since it will eventually let you edit the `slug` in the future that would make things clearer - otherwise how to know if the provided `slug` should be used to fetch the account or to update it?